packages/client-search/src/types/Rule.ts

```diff
@@ -125,6 +125,12 @@ export type ConsequenceParams = {
    * Behaves like optionalFilters.
    */
   readonly automaticOptionalFacetFilters?: readonly AutomaticFacetFilter[];
+
+  /**
+   * Content defining how the search interface should be rendered.
+   * A default value for this can be set via settings
+   */
+  readonly renderingContent?: Settings['renderingContent'];
```

Comment:
I probably missed it, but did we agree on `renderingContent` as a part of `Rule.Consequence`?

Reply:
In this object it's part of consequence.params.renderingContent, as we discussed